components/privatelink/signalr-private-link.tf

```
@@ -0,0 +1,13 @@
+data "local_file" "blob-private-link" {
+  filename = "${path.cwd}/../../environments/privatelink/signalr-private-link.yml"
+}
+
+module "blob-private-link" {
+  source              = "../../modules/azure-private-dns/"
+  cname_records       = yamldecode(data.local_file.blob-private-link.content).cname
+  a_recordsets        = yamldecode(data.local_file.blob-private-link.content).A
+  zone_name           = yamldecode(data.local_file.blob-private-link.content).name
+  vnet_links          = yamldecode(data.local_file.blob-private-link.content).vnet_links
```

needs updating to signalr
```suggestion
  cname_records       = yamldecode(data.local_file.signalr-private-link.content).cname
  a_recordsets        = yamldecode(data.local_file.signalr-private-link.content).A
  zone_name           = yamldecode(data.local_file.signalr-private-link.content).name
  vnet_links          = yamldecode(data.local_file.signalr-private-link.content).vnet_links
```